backend/package.json

```diff
@@ -9,52 +9,41 @@
     "PGLOCALINITCON": "postgresql://postgres@localhost:5432/"
   },
   "scripts": {
-    "local-drop-rebuild":
-      "yarn run drop-local-data;yarn run populate-local-schema;yarn run populate-local-data;yarn run build-local-schema;",
-    "local-drop-rebuild-with-tons-of-crossings":
-      "yarn run drop-local-data;yarn run populate-local-schema;yarn run add-tons-of-crossings-to-local-db;yarn run build-local-schema;",
+    "local-drop-rebuild": "yarn run drop-local-data;yarn run populate-local-schema;yarn run populate-local-data;yarn run build-local-schema;",
+    "local-drop-rebuild-with-tons-of-crossings": "yarn run drop-local-data;yarn run populate-local-schema;yarn run add-tons-of-crossings-to-local-db;yarn run build-local-schema;",
     "local-server": "yarn run local-drop-rebuild;yarn run postgraphql-local",
-    "local-server-with-tons-of-crossings":
-      "yarn run local-drop-rebuild-with-tons-of-crossings;yarn run postgraphql-local",
-    "rebuild-and-deploy":
-      "yarn run drop-data;yarn run populate-schema;yarn run populate-data;yarn run build-schema;sls deploy -v",
-    "rebuild-and-deploy-with-tons-of-crossings":
-      "yarn run drop-data;yarn run populate-schema;yarn run add-tons-of-crossings-to-db;yarn run build-schema;sls deploy -v",
+    "local-server-with-tons-of-crossings": "yarn run local-drop-rebuild-with-tons-of-crossings;yarn run postgraphql-local",
```

Comment:
We should probably make this start the XML server as well.

I see that we're using the postgrahql cli to launch the local server, which is different from what we do in production.

Maybe we could make a local server with a `/graphql` endpoint that calls our graphql handler lambda, and a `/xml` endpoint that calls our xml lambda.

Going a little further down that route, is it really necessary to use express in the xml lambda? We can add the `/xml` path through `serverless.yml`. Then the xml lambda can just be a simple function that queries postgres and not also an http server.

Reply:
I went ahead and got rid of express, I've added comments to https://github.com/cityofaustin/ctxfloods/issues/191 to update the local-server part of the backend to serve both endpoints in handler.js